locale/es/LC_MESSAGES/django.po

```diff
@@ -1870,6 +1893,18 @@ msgstr "opciones"
 msgid "editing"
 msgstr "editando"
 
+#: roleplay/templates/roleplay/race/race_confirm_delete.html:7
+#, fuzzy
+#| msgid "Delete"
+msgid "race delete"
+msgstr "borrar"
+
+#: roleplay/templates/roleplay/race/race_detail.html:52
+#, fuzzy
+#| msgid "Affected by armor"
+msgid "Affected by Armor"
+msgstr "afectado por la armadura"
```

Comment:
Change "Affected" to "affected by armor" to maintain lowercase

- [ ] Change to lowercase